humblegen/tests/rust/service-authorization-using-interceptor/spec.humble

```diff
@@ -0,0 +1,9 @@
+struct Post {
+    content: str
+}
+
+service BlogApi {
+    /// Must send header `Authorization: Custom AUTHZ_TOKEN`
+    /// otherwise authorization error.
+    POST /{user: str}/posts -> Post -> Post,
+}
```

Comment:
We should enforce a newline policy throughout the project...

Reply:
humblegenfmt!

Reply:
if you truly care about this, please open a new issue

Reply:
Don't care enough ;)